build/download-saturn-l2.js

```diff
@@ -58,7 +63,27 @@ async function main () {
         }
 
         const outFile = path.join(outDir, outName)
-        await pipeline(res.body, gunzip(), tar.extract(outFile))
+        if (match[3] === 'tar.gz') {
+          await pipeline(res.body, gunzip(), tar.extract(outDir))
+        } else {
+          // Darwin needs to be a zip for notarization
+          await mkdir(path.join(outDir, 'l2node-darwin-x64'))
+          const parser = unzip.Parse()
+          await Promise.all([
+            (async () => {
+              while (true) {
+                const entry = /** @type UnzipStreamEntry */ (await once(parser, 'entry'))
```

Comment:
There's a type error here, do you know how to fix this?

> Conversion of type 'any[]' to type 'Entry' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'any[]' is missing the following properties from type 'Entry': path, type, size, autodrain, and 54 more.ts(2352)

I tried

```js
/** @type UnzipStreamEntry */ /** @type unknown **/ (await once(parser, 'entry'))
```

But then there are type issues afterwards

Reply:
I think you need to put the type name into `{}`.

```suggestion
                const entry = /** @type {UnzipStreamEntry} */ (await once(parser, 'entry'))
```

Reply:
The documentation for `once` in Node typings says this:

> The `Promise` will resolve with an array of all the arguments emitted to the given event.

I added a commit to fix the types and then also another issue I discovered while running the script.

See [2fc3a2a](https://github.com/filecoin-station/filecoin-station/pull/197/commits/2fc3a2a6b8f2ce6220c69c419e763c4b6befdb78)